src/stories/Button.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Button from "../app/components/Shared/button";
+import '../app/styles/index.css';
```

Comment:
Instead of importing the styles into every story, you can use .storybook/preview.js for global code that applies to all stories. See: https://storybook.js.org/docs/riot/configure/overview#configure-story-rendering
So it can be imported only once.